### PR TITLE
add ci workflow for tool-package-quickstart

### PR DESCRIPTION
**Pull request by wangchao1230**
# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

# All Promptflow Contribution checklist:
- [ ] **The pull request does not introduce [breaking changes].**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [ ] **I have read the [contribution guidelines](../CONTRIBUTING.md).**
- [ ] **Create an issue and link to the pull request to get dedicated review from promptflow team. Learn more: [suggested workflow](../CONTRIBUTING.md#suggested-workflow).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: examples/tools/tool-package-quickstart/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+Install the package in your python env.
+```bash
+pip install -e .
+```
+
+## List tools
+
+List tools inside package.
+```bash
+pf tool list
```
Comment (wangchao1230) [inline examples/tools/tool-package-quickstart/README.md:15]: next PR: assert listed target tool

File: examples/tools/tool-package-quickstart/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+```
+
+## Validate tool
+
+Validate tool in a script file.
+```bash
+pf tool validate --source ./my_tool_package/tools/my_tool_1.py
```
Comment (16oeahr) [inline examples/tools/tool-package-quickstart/README.md:22]: What about the other tools? Do they need to be validated?

File: examples/tools/tool-package-quickstart/my_tool_package/tools/utils.py [hunk 3]
```diff
@@ -1,29 +1,17 @@
-from pathlib import Path
-import importlib.util
 from promptflow import PFClient
 
+package_name = "my_tool_package.tools"
```
Comment (16oeahr) [inline examples/tools/tool-package-quickstart/my_tool_package/tools/utils.py:3]: If the "my-tools-package" reverted to the previous version using yaml, how can we make sure this package_name still work?

Comment (chjinche) [reply examples/tools/tool-package-quickstart/my_tool_package/tools/utils.py:3]: I have the same concern. Looks this proposal has a special requirement for user: they need to place tool py in a separate folder from other files. Just wonder if we really need to have this limitation to users.

File: examples/tools/tool-package-quickstart/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+# pf tool validate --source ./my_tool_package/tools/tool_with_dynamic_list_input.py
```
Comment (wangchao1230) [inline examples/tools/tool-package-quickstart/README.md:23]: @lalala123123 could we validate all tools here?

**Pull request closed without merging**